lib/vagrant-service-manager/command.rb

```diff
@@ -221,6 +230,28 @@ def print_vagrant_box_version(script_readable = false)
           end
         end
       end
+
+      def restart_service(service)
+
+        if SCCLI_SERVICES.include? service
+          # since sccli understand k8s
+          service = "k8s" if service == "kubernetes"
+          # TODO : Handle the case where user wants to pass extra arguments to OpenShift service
+          command = "sudo sccli #{service}"
```

Comment:
I don't think this works for restarting services.  I don't think `sccli` has that concept yet.  @praveenkumar ?